.github/actions/label-closed-issues-with-date-v1/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "label-closed-issues-with-date-v1",
+  "version": "1.0.0",
+  "description": "A GitHub Action to check issue criteria and label closed issues with date information",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dequelabs/axe-api-team-public.git"
+  },
+  "author": "",
+  "license": "MPL-2.0",
+  "bugs": {
+    "url": "https://github.com/dequelabs/axe-api-team-public/issues"
+  },
+  "homepage": "https://github.com/dequelabs/axe-api-team-public#readme",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "ncc build src/index.ts --license ../../../licenses.txt",
+    "test": "mocha src/*.test.ts",
+    "coverage": "nyc npm run test",
+    "fmt": "prettier --write \"src/**/*.{js,ts,tsx,json,md,css,html}\"",
+    "lint": "eslint \"src/**/*.{js,ts,tsx}\""
+  },
+  "dependencies": {
+    "@actions/core": "^1.10.1",
+    "@actions/github": "^6.0.0"
+  },
```

Comment:
Can be removed - the dependencies are handled on the top level `package.json`. See another [action for example](https://github.com/dequelabs/axe-api-team-public/blob/main/.github/actions/add-to-board-v1/package.json)